.github/workflows/reusable.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     - name: Run Job 
       run: |
         set -e
-        docker run --rm -t --user $(id -u):$(id -g) \
+        docker run --gpus all --rm -t --user $(id -u):$(id -g) \
```

Comment:
It's ok now to pass in `--gpus all` , but might introduce errors when running on a machine without nvidia-docker installed.